CHANGELOG.md

```diff
@@ -1,19 +1,18 @@
 # Change log
 
 
-<!-- Template for documenting changes
+## jaxdecomp 0.0.1
```

Comment:
Are we really releasing a working code with several new additions as `v0.0.1` ?

I would personally prefer something like `0.1.0` and if we are going for https://semver.org/ they would even prefer changing the major version since some of the changes are not backward compatible (to my knowledge).
I know this is a big leap so I would go for 0.1 if you guys agree.

Reply:
I don't have enough open source experience to know the implication of a minor vs patch version
But I feel more like this was a minor version rather than a patch
(Then again the major version will always be 0 ..)